doc/releasenotes/14_0_0_summary.md

```diff
@@ -165,6 +186,30 @@ API endpoint `/throttler/throttle-app` now accepts a `ratio` query argument, a f
 - `1` means "always throttle"
 - any numbr in between is allowd. For example, `0.3` means "throttle in 0.3 probability", ie on a per request and based on a dice roll, there's a `30%` change a request is denied. Overall we can expect about `30%` of requests to be denied. Example: `/throttler/throttle-app?app=vreplication&ratio=0.25`
 
+API endpoint `/debug/vars` now exposes throttler metrics, such as number of hits and errors per app per check type. Example:
+
+```shell
+$ curl -s 'http://127.0.0.1:15100/debug/vars' | jq . | grep throttler
+  "throttler.aggregated.mysql.self": 133.19334,
+  "throttler.aggregated.mysql.shard": 132.997847,
+  "throttler.check.any.error": 1086,
+  "throttler.check.any.mysql.self.error": 542,
+  "throttler.check.any.mysql.self.total": 570,
+  "throttler.check.any.mysql.shard.error": 544,
+  "throttler.check.any.mysql.shard.total": 570,
+  "throttler.check.any.total": 1140,
+  "throttler.check.mysql.self.seconds_since_healthy": 132,
+  "throttler.check.mysql.shard.seconds_since_healthy": 132,
+  "throttler.check.vitess.error": 1086,
+  "throttler.check.vitess.mysql.self.error": 542,
+  "throttler.check.vitess.mysql.self.total": 570,
+  "throttler.check.vitess.mysql.shard.error": 544,
+  "throttler.check.vitess.mysql.shard.total": 570,
+  "throttler.check.vitess.total": 1140,
+  "throttler.probes.latency": 292982,
+  "throttler.probes.total": 1138
+```
+
```

Comment:
I don't think this belongs in v14.
The PR https://github.com/vitessio/vitess/pull/10443 was only merged into main yesterday and I don't see / expect a back port to v14.
What probably happened is that the summary file was changed when the PR was created (prior to release-14.0 branching), and was merged into main along with the code changes.
Can you double check this, and if necessary remove this section?
The other thing that needs to happen is that it may also need to be removed from the 14.0 release notes summary file in `main` and added to the 15.0 release notes file.

Reply:
> What probably happened is that the summary file was changed when the PR was created (prior to release-14.0 branching), and was merged into main along with the code changes.

Correct. Amending this today.

Reply:
Amendment: https://github.com/vitessio/vitess/pull/10572

Reply:
I see the metrics are already removed from `doc/releasenotes/14_0_0_summary.md` in `release-14.0`.

Reply:
I removed the entire paragraph.